ui/package-lock.json

```diff
@@ -5089,12 +5089,6 @@
         "entities": "^2.0.0"
```

Comment:
is there a rason there are changes to the package.lock in this PR but no corresponding changes to pckage.json

Reply:
it must've been because I added something to package.json during development and removed it, so package.lock changed. I'll revert the change to this file